community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/cache/NodeRelationshipCache.java

```diff
@@ -410,7 +412,7 @@ public int getCount( long nodeId, Direction direction )
         {
             this.base = base;
             assert chunkSize > 0;
-            this.array = arrayFactory.newDynamicByteArray( chunkSize,
+            this.array = arrayFactory.newDynamicByteArray( 1_000_000,
```

Comment:
it looks like we actually should use chunkSize here and it looks like that now it can only be equal to CHUNK_SIZE which is 1_000_000 anyway and no changes is needed here